salt/modules/chocolatey.py

```diff
@@ -741,19 +742,19 @@ def upgrade(name,
     if version:
         cmd.extend(['-version', version])
     if source:
-        cmd.extend(['-source', source])
+        cmd.extend(['--source', source])
     if salt.utils.is_true(force):
-        cmd.append('-force')
+        cmd.append('--force')
     if salt.utils.is_true(pre_versions):
-        cmd.append('-prerelease')
+        cmd.append('--prerelease')
     if install_args:
-        cmd.extend(['-installarguments', install_args])
+        cmd.extend(['--installarguments', install_args])
     if override_args:
-        cmd.append('-overridearguments')
+        cmd.append('--overridearguments')
     if force_x86:
-        cmd.append('-forcex86')
+        cmd.append('--forcex86')
     if package_args:
-        cmd.extend(['-packageparameters', package_args])
+        cmd.extend(['--packageparameters', package_args])
```

Comment:
https://chocolatey.org/docs/commands-install#options-and-switches

Not sure what you mean by triple? (The unix standard is typically a double hyphen, followed by a hyphen between each different word). With Chocolatey, we know folks are forgetful when it comes to command line arguments (myself included), so we try to be as forgiving as possible, as long as you get the spelling right.